docs/reference/tools.md

```diff
@@ -0,0 +1,67 @@
+# Tools used in this repo
+
+## Base tools
+
+These are helpful for everyone, and not cloud-specific.
+
+### [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+
+The canonical commandline tool for talking to kubernetes clusters.
+Debugging and understanding runtime behavior of our hubs is greatly
+enhanced by knowledge of how to use `kubectl`. Understanding how to
+use `kubectl` really helps understand how kubernetes itself works.
+
+#### Tips
+
+- Consider aliasing it to `k`, as you might be typing it a lot!
```

Comment:
That is a good idea although I would keep `kubectl` in our docs when we showcase commands...

Reply:
Yeah, I agree!